### ISSUE TITLE
'Contact the project team' with one member

### ISSUE THREAD

**Issue by edmundhighcock** [taigaio/taiga-back#1247]
**Do you want to request a *feature* or report a *bug*?**

Bug

**What is the current behavior?**

In a project with one member, click the 'Contact the project team' envelope icon triggers the following backtrace because it tries to send an email to an empty list of correspondants (because it tries to find all admin users who are not the person who clicked the button). I know it's not a particularly meaningful thing to do, but I was testing my email setup and it took me a while to work out why I was getting an error.

```
ERROR:2019-02-14 11:41:55,608: Internal Server Error: /api/v1/contact
taiga_1     | Traceback (most recent call last):
taiga_1     |   File "/usr/local/lib/python3.5/site-packages/django/core/handlers/exception.py", line 42, in inner
taiga_1     |     response = get_response(request)
taiga_1     |   File "/usr/local/lib/python3.5/site-packages/django/core/handlers/base.py", line 249, in _legacy_get_response
taiga_1     |     response = self._get_response(request)
taiga_1     |   File "/usr/local/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, [22/1808]
esponse
taiga_1     |     response = self.process_exception_by_middleware(e, request)
taiga_1     |   File "/usr/local/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
taiga_1     |     response = wrapped_callback(request, *callback_args, **callback_kwargs)
taiga_1     |   File "/usr/src/taiga-back/taiga/base/api/viewsets.py", line 106, in view
taiga_1     |     return self.dispatch(request, *args, **kwargs)
taiga_1     |   File "/usr/local/lib/python3.5/site-packages/django/views/decorators/csrf.py", line 58, in wrap
ped_view
taiga_1     |     return view_func(*args, **kwargs)
taiga_1     |   File "/usr/src/taiga-back/taiga/base/api/views.py", line 461, in dispatch
taiga_1     |     response = self.handle_exception(exc)
taiga_1     |   File "/usr/src/taiga-back/taiga/base/api/views.py", line 458, in dispatch
taiga_1     |     response = handler(request, *args, **kwargs)
taiga_1     |   File "/usr/src/taiga-back/taiga/projects/contact/api.py", line 43, in create
taiga_1     |     services.send_contact_email(self.object.id)
taiga_1     |   File "/usr/local/lib/python3.5/site-packages/celery/local.py", line 191, in __call__
taiga_1     |     return self._get_current_object()(*a, **kw)
taiga_1     |   File "/usr/local/lib/python3.5/site-packages/celery/app/task.py", line 379, in __call__
taiga_1     |     return self.run(*args, **kwargs)
taiga_1     |   File "/usr/src/taiga-back/taiga/projects/contact/services.py", line 48, in send_contact_email
taiga_1     |     email.send()
taiga_1     |   File "/usr/local/lib/python3.5/site-packages/django/core/mail/message.py", line 347, in send
taiga_1     |     return self.get_connection(fail_silently).send_messages([self])
taiga_1     |   File "/usr/local/lib/python3.5/site-packages/django/core/mail/backends/smtp.py", line 107, in send_messages
taiga_1     |     sent = self._send(message)
taiga_1     |   File "/usr/local/lib/python3.5/site-packages/django/core/mail/backends/smtp.py", line 120, in _send
taiga_1     |     recipients = [sanitize_address(addr, encoding) for addr in email_message.recipients()]
taiga_1     |   File "/usr/local/lib/python3.5/site-packages/django/core/mail/backends/smtp.py", line 120, in <listcomp>
taiga_1     |     recipients = [sanitize_address(addr, encoding) for addr in email_message.recipients()]
taiga_1     |   File "/usr/local/lib/python3.5/site-packages/django/core/mail/message.py", line 166, in sanitize_address
taiga_1     |     address = Address(nm, addr_spec=addr)
taiga_1     |   File "/usr/local/lib/python3.5/email/headerregistry.py", line 42, in __init__
taiga_1     |     a_s, rest = parser.get_addr_spec(addr_spec)
taiga_1     |   File "/usr/local/lib/python3.5/email/_header_value_parser.py", line 1993, in get_addr_spec
taiga_1     |     token, value = get_local_part(value)
taiga_1     |   File "/usr/local/lib/python3.5/email/_header_value_parser.py", line 1801, in get_local_part
taiga_1     |     print("VALUEL", len(value), value[0], file=f)
taiga_1     | IndexError: string index out of range
taiga_1     | [14/Feb/2019 11:41:55] "POST /api/v1/contact HTTP/1.0" 500 37

```

**If the current behavior is a bug, please provide the steps to reproduce.**

Create a project with one member, and click the  'Contact the project team' envelope icon in the project homepage.

**What is the expected behavior?**

An error message saying 'you are the only member!' or suchlike

**Is it happening in taiga.io or in your own instance?**

My own instance.

**What browser/version are you using?**

Firefox latest. Also taiga backend version 3.1.0

**Are there any console errors *(Ctrl + F12)* in red?**

XML Parsing Error: no element found
Location: https://xxxx/api/v1/userstories/18/watch
Line Number 1, Column 1:

Thanks for reporting!

**Comment by joskoanicic** [comment]
And if I may add, I am not sure if the intended feature _"tries to find all admin users who are not the person who clicked the button"_ is working either. I have tested it on multiple projects where there are multiple admins, and only one person is receiving the email not all admins as expected.

**Comment by migonzalvar** [comment]
This issue has been solved in 8229df3a8fc0d702807dd3feb490e4a6e0be7268. Please reopen if it's not the case.

**Comment by edmundhighcock** [comment]
Awesome thanks!
